site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -856,15 +856,14 @@ let them sign-in and start a new session.
 
 ### Call FedCM from within a cross-origin iframe
 
-FedCM can be invoked from within an iframe if the parent frame allows it with a
-permissions policy `identity-credentials-get`. All you have to do is to append
-an `allow="identity-credentials-get"` attribute to the iframe tag.
+FedCM can be invoked from within an iframe using an `identity-credentials-get` permissions policy, if the parent frame allows it. To do so, append
+the `allow="identity-credentials-get"` attribute to the iframe tag as follows:
```

Comment:
There's another way (preferable) to support FedCM in cross-origin iframe - using headers: https://docs.google.com/document/d/1LAjRuiGviuE-0bYDHErxzDjB1LBvhMv6XO2sEKHBfFY/edit#heading=h.g1uu0a58y99c

Reply:
That's described later in the section. I did so because it's the RP's action the IdP needs to ask them to do.

Reply:
Thanks! LGTM.